Stlying the experience section using a card layout similar to the interests section.

eriyezab opened the pull request.

eriyezab committed:
Change experience header to be centered and styled.

eriyezab committed:
Style experience section using navigating cards.

eriyezab committed:
Change project styling format to individual cards with pictures in
experience section.

eriyezab committed:
Merge branch 'master' of https://github.com/eriyezab/my-portfolio into ExperiencesStyling

eriyezab committed:
Add personal portfolio to project card deck.

eriyezab committed:
Move project images to bottom of card.

shrutip90 commented on portfolio/src/main/webapp/index.html:
```diff
@@ -131,33 +131,72 @@ <h5>My Favourite Songs</h5>
-                  gave product designers insight as to where potential clients were having trouble with the trial and steps that may need
-                  improving.</p>
+          <h2 class="display-3 text-center">Experience</h2>
+          <div class="card ">
```
Is this space intentional?

eriyezab replied:
No it is not. Fixed!

eriyezab committed:
Remove unnecessary space in index.html

ankitaj224 commented on portfolio/src/main/webapp/index.html:
```diff
@@ -131,33 +131,72 @@ <h5>My Favourite Songs</h5>
+                    <div class="tab-pane fade show active" id="work" role="tabpanel" aria-labelledby="work-tab">
+                      <div id="fivetran">
+                        <h4>Fivetran</h4>
+                        <p>I was a Product Analyst Intern at Fivetran from the beginning of June until the end of July in 2019. One of the
+                            projects I did there was to create a dashboard that illustrated new user retention across trial steps in a given month. This 
+                            gave product designers insight as to where potential clients were having trouble with the trial and steps that may need
+                            improving.</p>
```
Idk much about styling - But please verify if the indentation and spacing is as per Google Html/Css guide.

eriyezab replied:
https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping
https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns

In the first link it says to have at least 4 spaces for line wrapping but I think it refers to the element tags themselves because in the second link there are <p> tags that cross multiple lines but only use two spaces. Should I go ahead and change all my p tags that cross multiple lines to be indented with two spaces?

ankitaj224 replied:
Yes, we can follow that. Just  wanted us to be clear on the guidelines.

eriyezab committed:
Fix indentation of p tags and remove unneeded ones

HTML style guide showed an example p tag indented two spaces when covering multiple lines. Also removed optional <p> tags based off https://html.spec.whatwg.org/multipage/syntax.html#syntax-tag-omission

shrutip90 commented on portfolio/src/main/webapp/index.html:
```diff
@@ -88,33 +88,33 @@ <h3 class="text-center display-3">Interests</h3>
-                          enjoy watching the NBA and my favourite player would have to be Kevin Durant.</p>
+                        opportunity to play in University. Due to injuries, I decided not to continue playing 
+                        competetively but I still love playing for fun with friends and in Intramurals. I also
+                        enjoy watching the NBA and my favourite player would have to be Kevin Durant.
```
No \</p\> tag on this. Inside google code, I have seen that it is optional. Can you please check if its required and modify the whole file accordingly.
